### DISCUSSION TITLE
Process finished with exit code -1073741819 (0xC0000005)

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#14090] [Q&A]
rec_model_path=r"L:\paddle\models\openatom_rec_svtrv2_ch_infer",
                      rec_keys_path=r"L:\paddle\models\ch_PP-OCRv3_rec_infer\ppocr_keys_v1.txt",

**Comment by GreatV** [comment]
请提供更多信息

**Reply by nissansz** [reply]
是放在rapidocr rapid运行的。
